src/index.ts

```diff
@@ -1,15 +1,57 @@
+import { Document, getData } from "@govtechsg/open-attestation";
+import { get } from "lodash";
 import { getLogger } from "./util/logger";
+import { getProvider } from "./util/provider";
+import { getOwnerOf, transferTokenOwnership } from "./util/token";
+import { Issuer, TransferOwnership } from "./types";
 
-const { trace } = getLogger("sayHello");
+const { trace } = getLogger("TokenInfo:");
+
+interface TokenInterface {
```

Comment:
Do you really need this interface.

Reply:
not particularly. But good to have. Tells the method we are going to implement in out class.

Reply:
Okay, but the transforOwnership's return type wont be too useful if it's an object right?